### DISCUSSION TITLE
Adding the touche pad of PS4/PS5 as a touch pad or mouse

### DISCUSSION THREAD

**Discussion by zoqin** [AntiMicroX/antimicrox#449] [Ideas]
One of the best things of the DS is the touchpad it would be a great addition to AntiMicro especialy when I play with parsec on my phone, the precision of the touchpad is just somehting else

Thanks a lot for reading me

**Comment by pktiuk** [comment]
To be honest, I don't understand what do you mean.   
PS4 and PS5 gamepads have touchpads, but they are not used/mapped by antimicrox, because system recognizes them as a separate touchpad device (not part of gamepad) and this solution works very well.
